seatunnel-core/seatunnel-flink-starter/seatunnel-flink-15-starter/src/test/java/org/apache/seatunnel/core/starter/flink/multitable/MultiTableSinkTest.java

```diff
@@ -34,6 +35,7 @@
 import java.util.Collections;
 import java.util.List;
 
+@Order(1)
```

Comment:
Any reason for changed order?

Reply:
https://github.com/dailai/seatunnel/actions/runs/9015097694/job/24775117908

Because both use the InMemory sink. This may lead your test be failed.
<img width="937" alt="截屏2024-05-11 13 38 55" src="https://github.com/apache/seatunnel/assets/18141150/215a8c0f-ba81-416a-bd62-35aa86f5bb86">